.changeset/quiet-adults-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
```

Comment:
Overall, I think this is right. Some of the TS is a little "big-brain" for me to understand and it looks like you might still be tidying up a little bit, but I think it's mostly good to go.

My concern/observation: this does change the _behavior_ of `emptyWidgets`. Before we only enabled the "Check" button after a deep validation check (including the validation that requires the answers). Now we'll enable the "Check" button after a shallow validation check (only checking if the answer is scorable). 

This is inevitable and baked into the project. It just occurs to me that we haven't really shared this with content and support advocates. I think it'll hardly be noticeable, but what do I know.